Modules/_ctypes/_ctypes.c

```diff
@@ -489,6 +490,7 @@ ctype_clear_stginfo(StgInfo *info)
     Py_CLEAR(info->restype);
     Py_CLEAR(info->checker);
     Py_CLEAR(info->module);
+    Py_CLEAR(info->pointer_type);
```

Comment:
```suggestion
    Py_CLEAR(info->pointer_type);
    Py_CLEAR(info->module);  // decref the module last
```
Could you move down `Py_CLEAR(info->module)` like the sample to ensure that `module_free()` is invoked after all infos are cleared?

Reply:
Thanks! I think I have to move `pointer_type` declaration too.